cli/cmd/sync_test.go

```diff
@@ -58,29 +58,6 @@ func TestSync(t *testing.T) {
 		{
 			name:   "multiple_destinations",
 			config: "multiple-destinations.yml",
-			summary: []syncSummary{
```

Comment:
I believe this worked before since all tests were sharing the `.cq` directory which means they also share `summaryPath`.
If you check `multiple-destinations.yml` you'll see the source version is different than listed here

Reply:
Also had to do https://github.com/cloudquery/cloudquery/pull/17320/commits/206bbaf7190d5a7aeda3ddc59b545716d741b7db to fix other sync summary tests due to the re-use of the output path